README.md

```diff
@@ -140,8 +140,8 @@ settings = MySettings(modules=[my_module, my_awesome_module], dotenv_path='/path
 # PSYDUCK = 'yaml'
 ```
 
-Временное переопределение Property
-----------------------------------
+Temporary Property override
+---------------------------
 _my_module.py_
```

Comment:
[MD036 - Emphasis used instead of a heading](https://github.com/DavidAnson/markdownlint/blob/v0.13.0/doc/Rules.md#md036)